webhook.rst

```diff
@@ -84,8 +84,17 @@ Mailer service  Parser service name
 =============== ==========================================
 Mailgun         ``mailer.webhook.request_parser.mailgun``
 Postmark        ``mailer.webhook.request_parser.postmark``
+Mailjet         ``mailer.webhook.request_parser.mailjet``
+Brevo           ``mailer.webhook.request_parser.brevo``
+Vonage          ``mailer.webhook.request_parser.vonage``
+Sendgrid        ``mailer.webhook.request_parser.sendgrid``
```

Comment:
Can you please rebase, and add versionadded directives for 6.3 too?


Also Bravo, Vonage and Mallet needs to get a `yes` in the mailer.rst file in webhook column

Reply:
Vonage is not present in Mailer. But I added for the others

Reply:
Vonage is part of the Notifier. Should we mention webhook support there as well?

Reply:
I'd say yes! I reworded the sentence above the table to remove "mailer". What do you think?

Reply:
Sounds good, but then also remove "Mailer" from the table header. ^_^

Reply:
Removed :+1: